tests/common.go

```diff
@@ -8198,6 +8209,86 @@ func StartTorpedoTest(testName, testDescription string, tags map[string]string,
 		RunIdForSuite = testrailuttils.AddRunsToMilestone(testRepoID)
 		CurrentTestRailTestCaseId = testRepoID
 	}
+	log.Infof("TOGGLE_PURE_MGMT_IP: %v", os.Getenv("TOGGLE_PURE_MGMT_IP"))
```

Comment:
Can we extract the giant if and else to separate functions so that we don't break up the flow too much? "If a block is taller than your screen, extract it to a function"

Reply:
Extracted the entire code block into two different functions